glslc/src/main.cc

```diff
@@ -307,6 +308,8 @@ int main(int argc, char** argv) {
       }
     } else if (arg == "-fauto-bind-uniforms") {
       compiler.options().SetAutoBindUniforms(true);
+    } else if (arg == "-fupgrade-textures") {
```

Comment:
I found the original commit of this feature in Glslang.
https://github.com/KhronosGroup/glslang/pull/856

And I see you're adding this to the glslangValidator command line via https://github.com/KhronosGroup/glslang/pull/2616 

Can I suggest -fauto-combined-image-sampler instead?  That would also be more in line with what I suggested for glslangValidator.

Also, please rename the other methods and state variables to match.

Reply:
Also, please update the glslc help string. I suggest:


  -fauto-sampled-textures    Removes sampler variables and converts existing textures to combined image-sampler